### PR TITLE
SIAC-100: Devlab configuration improvements.                            ...

### PR DESCRIPTION
**Pull request by plomakin**
Fixed mysql parameters in configuration files for devlab:
* mysql_hosts for src and dst
* mysql_dump file parameter
* small fixes in pip requirements list

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/28/

**Comment by miarmak** [comment]
Looks good. +1 from me.
But it depends on #159, so do not merge before it please.

**Comment by oleksii-shyman** [comment]
Looks good. +1 from me.
But it depends on #159, so do not merge before it please.

**Comment by plomakin** [comment]
No problem

**Comment by Wasapl** [comment]
Merged build finished. Test FAILed.   http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/29/

**Comment by Wasapl** [comment]
Merged build finished. Test FAILed.   http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/30/

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/31/

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
---allow-external mysql-connector-python
+mysql-connector-python
```
Comment (miarmak) [inline requirements.txt]: Could you put it in the alphabetical order please?

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/35/

**Comment by miarmak** [comment]
Thanks! +1

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**